### ISSUE TITLE
config file

### ISSUE THREAD

**Issue by asw91666** [garvita-tiwari/PoseNDF#7]
First, thanks for the nice work you've publicly released!

But, your code is not matched with your config file, "configs/amass.yaml" when following your README.md
especially, "python trainer.py --config=configs/amass.yaml", "python trainer.py --config=configs/amass.yaml --test".

Could you share the corrected one?

**Comment by HospitableHost** [comment]
same problem

**Comment by tommyshelby4** [comment]
There is no entry DFNet in the dictionary loaded from amass.yaml so you need to update it, same problem.

**Comment by HospitableHost** [comment]
> There is no entry DFNet in the dictionary loaded from amass.yaml so you need to update it, same problem.

And the pretrained model is not correct.

**Comment by tommyshelby4** [comment]
Indeed

**Comment by HrishavBakulBarua** [comment]
Can anyone comment on the training? The pre-trained models are not working.

**Comment by garvita-tiwari** [comment]
> First, thanks for the nice work you've publicly released!
> 
> But, your code is not matched with your config file, "configs/amass.yaml" when following your README.md especially, "python trainer.py --config=configs/amass.yaml", "python trainer.py --config=configs/amass.yaml --test".
> 
> Could you share the corrected one?

Please pull the updated code and try again.

**Comment by garvita-tiwari** [comment]
> Can anyone comment on the training? The pre-trained models are not working.

Please pull the updated code and model and try again.

**Comment by HospitableHost** [comment]
> > Can anyone comment on the training? The pre-trained models are not working.
> 
> Please pull the updated code and model and try again.

There is no file in the link "pretrained model" .

**Comment by tommyshelby4** [comment]
The DFNet still does not exist in the amass.yml. I pulled the code. I get the same error.   
![image](https://user-images.githubusercontent.com/36934521/214335780-f0743cde-a569-42bc-97f5-eb926d16eab5.png)

**Comment by HospitableHost** [comment]
> The DFNet still does not exist in the amass.yml. I pulled the code. I get the same error. ![image](https://user-images.githubusercontent.com/36934521/214335780-f0743cde-a569-42bc-97f5-eb926d16eab5.png)

you should modify the yaml file.

**Comment by tommyshelby4** [comment]
Modify it in which way? Could you help? I guess I should specify an DFNet entry in the amass.yaml dictionary with keys **in_dim** and **dim**, but what should the values?
